intro_modern.tex

```diff
@@ -129,14 +130,14 @@ \subsection{About keywords}
 
 \section{Notations for grammars}
 
-In this document, grammar rules are given in BNF form. In grammar
-rules, we use extra notations $e^*$ to denote repetition of zero, one
+In this document, grammar rules are given in BNF form. In the grammar
+rules, we use the extra notations $e^*$ to denote repetition of zero, one
 or more occurrences of $e$, $e^+$ for repetition of one or more
 occurrences of $e$, and $e^?$ for zero or one occurrence of $e$.  For
 the sake of simplicity, we only describe annotations in the usual
 \lstinline|/*@ ... */| style of comments. One-line annotations
-in \lstinline|//@| comments are alike.
-
+in \lstinline|//@| comments are similar.
+\marginpar{Somewhere it needs to be defined in what way ACSL specifications can be split across consecutive \lstinline|//@| comments, if they can be split at all. For example, JML requires each function contract clause to be completely within an annotation.}
```

Comment:
The current implementation in Frama-C even requires the entire contract to be in the same comment. I'll propose something in this regard.